templates/quiz/start-button.php

```diff
@@ -31,6 +32,8 @@
 
 	<?php if ( $quiz ) : ?>
 
+		<?php $start_button_string = $quiz->can_be_resumed() && $quiz->can_be_resumed_by_student() ? __( 'Restart Quiz', 'lifterlms' ) : __( 'Start Quiz', 'lifterlms' ); ?>
```

Comment:
I think we have to only check if the current user can resume the last attempt, so only:
`$quiz->can_be_resumed_by_student()`
because it depends on whether the attempt can be resumed not on whether the quiz's "Can be resumed" option is set or not, as it is not retroactive.

Reply:
done and understood.